File: minecraft/origin/src/main/java/com/github/theword/queqiao/event/minecraft/MinecraftPlayerChatEvent.java
package com.github.theword.queqiao.event.minecraft;

import com.github.theword.queqiao.tool.event.base.BasePlayerChatEvent;

public class MinecraftPlayerChatEvent extends BasePlayerChatEvent {
    public MinecraftPlayerChatEvent(String messageId, MinecraftPlayer player, String message) {
        super("MinecraftPlayerChatEvent", messageId, player, message);
    }
}

File: minecraft/origin/src/main/kotlin/com/github/theword/queqiao/event/minecraft/MinecraftPlayerChatEvent.kt
package com.github.theword.queqiao.event.minecraft

import com.github.theword.queqiao.tool.event.base.BasePlayerChatEvent

data class MinecraftPlayerChatEvent(
    private val messageId: String = "", private val player: MinecraftPlayer, private val message: String
) : BasePlayerChatEvent("MinecraftPlayerChatEvent", messageId, player, message)
